synapse/models/inet.py

```diff
@@ -2397,16 +2374,15 @@ async def _onSetWhoisText(node, oldv):
                     'doc': 'The FQDN within the dst URL (if present).'}),
             )),
 
-            # FIXME used anywhere?
             ('inet:url:mirror', {}, (
+
```

Comment:
Line 2359, rename inet:urlredir > inet:url:redir per discussion

Reply:
This was from a subsequent discussion and is in the next delta